src/helpers.js

```diff
@@ -70,7 +70,7 @@ export function filterRedundantErrors(root, parent, key) {
   );
 }
 
-export function createErrorInstances(root) {
+export function createErrorInstances(root, indent) {
```

Comment:
Why do we need to pass `indent` here? We use it only inside `print` right? Or, are you using it elsewhere?

Reply:
I use it when I create error instances, i.e. `new AdditionalPropValidationError` etc. I need to pass `indent` setting into these constructors somehow.